src/cpp/src/llm_pipeline.cpp

```diff
@@ -139,8 +139,8 @@ ov::genai::LLMPipeline::LLMPipeline(
         m_pimpl = std::make_unique<ContinuousBatchingAdapter>(models_path, tokenizer, scheduler_config, device, device_properties);
     }
 
-    if (m_pimpl == nullptr && device == "NPU") {
-        m_pimpl = static_llm::LLMPipelineFactory::create(models_path, tokenizer, device, properties);
+    if (m_pimpl == nullptr && device == "NPU" && properties.count("STATIC_PIPELINE")) {
```

Comment:
Following my previous review - can we just get rid of `STATIC_PIPELINE` since our only option is `STATEFULL`?

Reply:
Most likely it's not the case, but what if we have another static pipeline implementation? I plan to remove it anyway next week preferably when switching to generic pipeline doesn't bring any perf regressions